prod_api/content_types/service.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2025 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from ..service import ProdApiService
+import bson
+import re
+import superdesk
+
+
+class ContentTypesService(ProdApiService):
+    excluded_fields = ProdApiService.excluded_fields
+
+    def get_output_name(self, profile):
+        try:
+            _id = bson.ObjectId(profile)
+            item = superdesk.get_resource_service("content_types").find_one(req=None, _id=_id) or {}
+            name = item.get("output_name") or item.get("label", str(_id))
```

Comment:
would be better to update logic in https://github.com/superdesk/superdesk-core/blob/develop/apps/content_types/content_types.py#L239-L245 and use that method here instead of copying it

Reply:
because when I tried to import functionality from the `superdesk_app`, I got some errors like `key_error`, so that's why I added a new service in the `prod_api` related to `content_types`.

Reply:
would be good to reuse that to avoid that code duplication. not sure if we were doing so far

Reply:
I tried to use this superdesk service here, but it's not working; may be I am missing something.

Reply:
what exactly is not working when using the `ContentTypesService` ?

Reply:
it would be also an option to put that code without `find_one` into some helper method and reuse that one at least

Reply:
actually its gives us a keyerror, so we have to define a new `ContentTypesService` in the `production_api` app.

Reply:
what keyerror exactly

Reply:
So when we try to use a superdesk api `content_types` service, it fetches this `prod_api` service.
and when I tried to change the name of the `prod_api` service, it gave `None` in this case

Reply:
when you add it to `INSTALLED_APPS` ?